pages/index.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <div class="container">
+    <h1 class="title">Pokemon Familiy</h1>
```

Comment:
Do we need the css class here? can't we just target h1 since there should only be 1 h1 in a page?